pkg/transformer/combiner.go

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package transformer
+
+import (
+	"fmt"
+	mysql2 "github.com/arana-db/arana/pkg/constants/mysql"
+	"github.com/arana-db/arana/pkg/mysql"
+	"github.com/arana-db/arana/pkg/proto"
+	ast2 "github.com/arana-db/arana/pkg/runtime/ast"
+	"github.com/pkg/errors"
+	"strconv"
+)
+
+type combinerManager struct {
+}
+
+type (
+	Combiner interface {
+		Merge(result proto.Result, loader *AggrLoader) (proto.Result, error)
+	}
+)
+
+func (c *combinerManager) Merge(result proto.Result, loader *AggrLoader) (proto.Result, error) {
```

Comment:
Use `func (c combinerManager)` instead, because it's an empty struct, no need to use pointer.

Reply:
address